docs/docusaurus/static/_redirects

```diff
@@ -250,4 +250,18 @@
 /docs/guides/expectations/advanced/how_to_create_a_new_expectation_suite_by_profiling_from_a_jsonschema_file /docs/guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant
 
 # removing yml-driven datacontext in favor of ephemeral
-/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file /docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context
\ No newline at end of file
+/docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file /docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/how_to_explicitly_instantiate_an_ephemeral_data_context
+
+# Redirects for Contributing docs
+
+/docs/contributing/contributing_setup https://github.com/great-expectations/great_expectations/blob/contributing-and-style-readme/CONTRIBUTING_CODE.md
+/docs/contributing/contributing_checklist /docs/contributing/contributing
+/docs/contributing/contributing_github /docs/README
```

Comment:
I tested this redirect in the netlify preview and it does not appear to be working.